packages/admin-stories/package.json

```diff
@@ -23,9 +24,11 @@
     "dependencies": {
         "apollo-link-rest": "^0.8.0-beta.0",
         "graphql-anywhere": "^4.2.7",
-        "qs": "^6.9.1"
+        "qs": "^6.9.1",
+        "cli-progress": "^3.8.2"
```

Comment:
Why do we need this dependency in the stories?

Reply:
so true, they are already gone in https://github.com/vivid-planet/comet-admin/pull/344/commits/095c1ce15bf75ebe5d472387a3ce06c2f0a82eef